src/viam/app/data_client.py

```diff
@@ -461,6 +469,7 @@ async def export_tabular_data(
             DataClient.TabularDataPoint(
                 part_id=resp.part_id,
                 resource_name=resp.resource_name,
+                resource_subtype=resp.resource_subtype,
```

Comment:
given above comment re: property decorator, I don't think we need to (or should) do this setting here.